tools-v2/docs/zh/develop.md

```diff
@@ -293,6 +301,19 @@ make
 
 生成的二进制文件保存为 `tools-v2/sbin/curve`。
 
+> 若出现诸如没有 `protoc-gen-go` 、`protoc-gen-go-grpc`，但在上面步骤已经install的情况，可查看 `GOROOT` 下（一般为 `~/go` ）是否有二进制文件，若有，可以将其拷贝至go的安装路径后再重新编译。
+>
+> ```shell
+> cd ~/go/bin
+> cp ./protoc-gen-go* /usr/local/go/bin
+> ```
+>
+> 若出现musl-gcc相关的报错，可进行如下操作直接编译：
```

Comment:
You can also give the installation documentation:
https://command-not-found.com/musl-gcc

Reply:
done